### PR TITLE
Constants

### PR DESCRIPTION
**Pull request by guillaume-gomez**
https://github.com/guillaume-gomez/NodeRace/issues/16

### CODE REVIEW

**Comment by guillaume-gomez** [comment]
Doing this issue shows new mistakes during the development.
I will write other issues(and write code of course!) to fix the mess i did (bouhhhh)

File: server/app.js [hunk 1]
```diff
@@ -13,11 +13,16 @@ var gameEngine = require('./js/engine');
...
 //fonction pour la gestion de chat
 var chatF = require('./js/chat');
-
-
 // Chargement de socket.io
 var io = require('socket.io')(server);
 
+//load constants in server and client
+app.use(express.static('public'));
+app.use('/public', express.static('public'));
```
Comment (AdrienJarretier) [inline server/app.js:20]: I believe you can remove this line, already set to use public just before

File: client/js/game.js [hunk 2]
```diff
@@ -62,15 +62,15 @@ function Game (socket, myId)
...
-        socket.on('counting', function(count) {
+        socket.on(jaws.constants.counting, function(count) {
         	console.log("console : couting "+count);
         	m_counting = count;
         	m_hubTxt.text = m_counting;
         });
 
         socket.on(jaws.constants.endGame, function(message) {
         	m_hubTxt.text = message;
-        	console.log("finde partieserveur bitch");
```
Comment (AdrienJarretier) [inline client/js/game.js:73]: i kinda liked this log x)

File: server/public/constants.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  Constants.prototype.connection = "connection";
+  Constants.prototype.closeConnection = "closeConnection";
+  Constants.prototype.endGame = "endGame";
+  Constants.prototype.counting = "counting";
+  Constants.prototype.startGame = "startGame";
+  Constants.prototype.login = "login";
+  Constants.prototype.logins = "logins";
+  Constants.prototype.ping = "ping";
+  Constants.prototype.trackPosition = "trackPosition";
+  Constants.prototype.gameDeconnexion = "gameDeconnexion";
```
Comment (AdrienJarretier) [inline server/public/constants.js]: gameDisconnection
or
gameDisconnect

File: server/public/constants.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  Constants.prototype.myPosition = "myPosition";
+  Constants.prototype.position = "position";
+  Constants.prototype.isExist = "isExist";
+  Constants.prototype.closeCo = "closeCo";
+  Constants.prototype.deconnexion = "deconnexion";
```
Comment (AdrienJarretier) [inline server/public/constants.js]: disconnection / disconnect but this would duplicate with line 25

File: server/public/constants.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  Constants.prototype.instanceDeconnexion = "instanceDeconnexion";
```
Comment (AdrienJarretier) [inline server/public/constants.js]: again disconnect / disconnection

File: server/public/constants.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  Constants.prototype.disconnect = "disconnect";
+
+  Constants.prototype.acceleration = "acceleration";
+
+  Constants.prototype.switchTrack = "switchTrack";
+
+  Constants.prototype.message = "message";
+  Constants.prototype.oldMessages = "oldMessages";
+
+  Constants.prototype.messageServeur = "messageServeur";
```
Comment (AdrienJarretier) [inline server/public/constants.js]: -> serverMessage

**Comment by AdrienJarretier** [comment]
I hilighted constants that need to be set in proper english
We should order them alaphabetically it would be easier to read and use
also I think there are either duplicates and/or ambiguous names

**Pull request merged**